src/entities.test.js

```diff
@@ -0,0 +1,32 @@
+const entities = require('./entities')
+const assert = require('assert')
+
+describe('entities', () => {
+    it('should create an empty list', () => {
+        assert.deepEqual(entities.all, new Map())
+    })
+
+    it('should throw an error when no id is provided and the entity has no name', () => {
+        const entityWithNoName = {}
+        assert.throws(() => entities.add(entityWithNoName), /^Error: Herbarium requires an id for this entity or an entity with a non empty name.$/)
+    })
+
+    it('should create an item with the provided id and entity', () => {
+        const anEntity = {}
```

Comment:
would be possible to use the actual herbs entity implementation?

Reply:
Yes, but there would be necessary to add the dev-dependency of `@herbsjs/gotu`.

Reply:
I think it would be ok for `devDependencies` to have `@herbsjs/herbs`